test/tests/providers/capi/capa/platform/atlas/alerting-rules/mimir.rules.test.yml

```diff
@@ -570,10 +570,10 @@ tests:
   # Test for MimirObjectStorageLowRate alert
   - interval: 1m
     input_series:
-      - series: 'cortex_bucket_store_sent_chunk_size_bytes_count{cluster_id="myinstall", cluster_type="management_cluster", installation="myinstall", namespace="mimir", pipeline="stable", provider="capa"}'
-        values: "_x90 1+1x90 90+0x200"
-      - series: 'capi_cluster_status_condition{cluster_id="myinstall", cluster_type="management_cluster", installation="myinstall", namespace="mimir", pipeline="stable", provider="capa", name="myinstall", type="ControlPlaneReady", status="True"}'
-        values: "1+0x380"
+      - series: 'thanos_objstore_bucket_operation_failures_total{cluster_id="myinstall", cluster_type="management_cluster", component="store-gateway", installation="myinstall", namespace="mimir", pipeline="stable", provider="capa"}'
```

Comment:
You removed the `_x90` test, which checked the behaviour when the counters don't exist. This is usually what happens when no data has been sent since startup.
I'm thinking that may actually the specific case that was paging in the original issue.

Are we confident these counters always exist?
Does the `thanos_objstore_bucket_operations_total` counter exist if no data was successfully sent?
Does the `thanos_objstore_bucket_operation_failures_total` counter exist if there were no failures?
What does the rule do if one of these counters does not exist?

Reply:
Let's pair tomorrow because my brain does not process 😂

Reply:
So from this on gaggle:

![image](https://github.com/user-attachments/assets/9b11bea0-06f9-4515-bb15-cea5466c7efa)

The query itself pages whether we keep the unless or not (so this was not related to missing metrics) at all

The 2 metrics in question have had no missing metrics in the last 3 months but how confident do you need me to be that they are there on all clusters?